alchemy.lic

```diff
@@ -0,0 +1,164 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#-------------
+=end
+
+custom_require.call(%w[common common-crafting common-items common-travel common-arcana])
+
+class Alchemy
+  include DRC
+  include DRCC
+  include DRCI
+  include DRCT
+  include DRCA
+
+  def initialize
+    arg_definitions = [
+      [
+        { name: 'target', regex: /\w+/, description: 'Ingredient or recipe.' },
+        { name: 'forage', regex: /forage/i, optional: true },
+        { name: 'prepare', regex: /prepare/i, optional: true },
+        { name: 'craft', regex: /craft/i, optional: true },
+        { name: 'herb_quantity_needed', regex: /\d+/, optional: true, description: 'Used to calculate required herbs to forage and process.' },
+        { name: 'debug', regex: /debug/i, optional: true }
+      ]
+    ]
+
+    args = parse_args(arg_definitions)
+
+    echo "*** incorrect arguments ***" if args.craft and (args.forage or args.prepare)
```

Comment:
💡 Prefer `&&` and `||` to `and` and `or`

Reply:
Sorry, was Powers logic and was working so I just skipped over. Will change.